### DISCUSSION TITLE
Unable to use oomph setup configuration on 19.02.2024

### DISCUSSION THREAD

**Discussion by peterkir** [eclipse-oomph/oomph#69] [General]
Hi all,

When trying to use oomph setup configuration via todays eclipse-installer via following command
`eclipse-inst-jre-win64.exe https://raw.githubusercontent.com/eclipse-oomph/oomph/master/setups/configurations/OomphConfiguration.setup`
I get the following error in Progress page 

1. for `P2 directory` task
  ```
ERROR: org.eclipse.equinox.p2.director code=0 Software being installed: artificial_root 1.0.0.v1708343617262
  ERROR: org.eclipse.equinox.p2.director code=0 Missing requirement: artificial_root 1.0.0.v1708343617262 requires 'org.eclipse.equinox.p2.iu; org.eclipse.mylyn.commons.activity.feature.group 0.0.0' but it could not be found
```

2. for `Modular Target` task

`No repository found at https://download.eclipse.org/eclipse/updates/4.29-I-builds.`

Any hints how to fix this?

**Comment by merks** [comment]
If I try this configuration, the p2 task looks like this on the Confirmation page:

![image](https://github.com/eclipse-oomph/oomph/assets/208716/50b28497-e68d-456b-babe-a8a6a296a176)

Nothing is using 4.29-I-builds which hasn't existed in a very long time.

Perhaps remove all contents from <home>/.eclipse/org.eclipse.oomph.setup/cache because maybe you have some very old cached content?

**Comment by merks** [comment]
Another thing to consider is if you are overwriting an existing installation and there is an old git clone there that has not been pulled in a long time?

**Comment by peterkir** [comment]
Tried it on MacOSX. Worked perfectly fine. Will try again on the other two windows machines.
Those contained complete self-hosted index and setup models.
This copied content is very old. So thanks for the feedback. Problem is soved.

**Reply by merks** [reply]
Thanks for asking a question and for providing feedback on the answer. 🙂
